src/SFA.DAS.Admin.Aan.Web.UnitTests/Controllers/ManageEvent/CheckYourAnswersControllerTests.cs

```diff
@@ -90,13 +90,26 @@ public class CheckYourAnswersControllerTests
         sessionServiceMock.Verify(x => x.Set(It.Is<EventSessionModel>(s => s.HasSeenPreview == true)), Times.Once());
     }
 
-    [Test, MoqAutoData]
-    public void GetCheckYourAnswers_HasSeenPreviewTrue_NoUpdateToSessionModel(
-        [Frozen] Mock<IOuterApiClient> outerAPiMock,
-        [Frozen] Mock<IValidator<CheckYourAnswersViewModel>> validatorMock,
-        List<Calendar> calendars,
-        GetRegionsResult regionsResult)
+    [TestCase(true, true, false)]
+    [TestCase(true, false, true)]
+    [TestCase(false, true, true)]
+    [TestCase(false, false, true)]
+    public void GetCheckYourAnswers_SeenPreviewAndDirectCallSettings_CheckUpdateToSessionModel(bool hasSeenPreview, bool directCallFromCheckYourAnswers, bool callsToSetEventSessionModel)
```

Comment:
What does direct call settings mean?

Reply:
Some stories needed a way of detecting whether the call came directly from the check your answers page, or via a different page alongside has preview been seen, as if these are both true, there is no need to Set the SessionEventModel for these parameters. The parameter name on the session model is actually 'DirectCallFromCheckYourAnswers'.  I will rename this test  on the next story (CSP-1002)